src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -323,6 +338,15 @@ private Boolean structureStep(final Structure.StructureBlock structureBlock)
                 return false;
             }
 
+            // we need a shovel to transform dirt into grass path
+            if (block instanceof BlockGrassPath)
+            {
+                if (!holdEfficientTool(block))
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1066)